containers/podman-desktop
# chore: remember last child page visited for kubernetes and preferences pages

### What does this PR do?
This PR adds stores to keep track of the last sections visited in submenu pages, so that when leaving and coming back, the user will come back to the last section they visited in the same submenu (can check with Kubernetes for now)

### Screenshot / video of UI

[Screencast from 2024-10-21 00-38-16.webm](https://github.com/user-attachments/assets/906d4488-8b33-41e3-a2c1-95a991d2a9bd)




### What issues does this PR fix or reference?
Closes https://github.com/containers/podman-desktop/issues/9057



### How to test this PR?
Enter the Kubernetes (nodes) page, go to one of the other pages in Kubernetes (e.g. Deployments), exit to some other page in PD that is not in Kubernetes, and then go back to Kubernetes. You should be back in your last visited section page (e.g. Deployments)
